Webmin/input/checks/accounts_passwd_cmd.xml

```diff
@@ -0,0 +1,33 @@
+<def-group>
+  <definition class="compliance" id="accounts_passwd_cmd" version="1">
+    <metadata>
+      <title>Use passwd for changing passwords.</title>
+      <affected family="unix">
+        <product>Webmin</product>
```

Comment:
On 2/25/15 2:46 PM, thenefield wrote:
>
> In Webmin/input/checks/accounts_passwd_cmd.xml 
> <https://github.com/OpenSCAP/scap-security-guide/pull/457#discussion_r25372073>:
>
> > @@ -0,0 +1,33 @@
> > +<def-group>
> > +  <definition class="compliance" id="accounts_passwd_cmd" version="1">
> > +    <metadata>
> > +      <title>Use passwd for changing passwords.</title>
> > +      <affected family="unix">
> > +        <product>Webmin</product>
>
> This will be a bit tedious. I have typically been testing with the 
> tip. But your right. Won't be done overnight, but I will look into 
> validating when each needed feature is introduced and reference said 
> version in each rule.
>

Supporting every version of Webmin would likely be a time sink. For the 
initial commit, lets start with whatever version you're using now (e.g. 
1.730).

There's nothing wrong with drawing a line on where SCAP development 
starts. If people need historical version support, they'll send some 
EMails or perhaps generate patches (e.g. your RHEL5 content!).